### ISSUE TITLE
Timeline not loading

### ISSUE THREAD

**Issue by WindHun7er** [jonjomckay/fritter#82]
As said in the title, when I want to access the "all" group, it starts to load and then after a few seconds the wheel spins indefinitely. 

I did a logcat and I get the following error, I don't know if this helps: 

```
2021-05-13 17:49:47.584 3144-6405/? E/flutter: [ERROR:flutter/lib/ui/ui_dart_state.cc(186)] Unhandled Exception: Connection closed before full header was received
    #0      IOClient.send (package:http/src/io_client.dart:61)
    <asynchronous suspension>
    #1      BaseClient._sendUnstreamed (package:http/src/base_client.dart:93)
    <asynchronous suspension>
    #2      _withClient (package:http/http.dart:164)
    <asynchronous suspension>
    #3      _FritterTwitterClient.fetch (package:fritter/client.dart:86)
    <asynchronous suspension>
    #4      Future.timeout.<anonymous closure> (dart:async/future_impl.dart:809)
    <asynchronous suspension>
    #5      _FritterTwitterClient.get.<anonymous closure> (package:fritter/client.dart:34)
    <asynchronous suspension>
```

(I also point out that I have several hundred subscriptions and this does not happen or rarely in groups where I have only a few dozen subscriptions).

**Comment by jonjomckay** [comment]
I feel like this might be an issue with the way Fritter is loading the tweets in parallel - I'd guess that too many requests are being fired off at once. I'll test with a few hundred subscriptions and see what happens!

**Comment by WindHun7er** [comment]
Yeah, I though the same ! 
Especially, when I said I had "several" hundred subscriptions, it was around 721 to be more precise...!  So, I admit that's probably way too many requests at once... ˆˆ' (but that corresponds to all the accounts I was following on my twitter account...)

**Comment by jonjomckay** [comment]
@WindHun7er can you confirm if this is fixed in the latest version (v2.6.2)?

**Comment by WindHun7er** [comment]
Unfortunately not. I simply get the following error now. I tried to create another group but it gives the same thing as before. Only groups with a few dozen subscriptions work.

![Screenshot_20210527-214408_Fritter](https://user-images.githubusercontent.com/80457123/119888172-54b12700-bf35-11eb-88b9-7f0eed3ec9c3.jpg)

**Comment by jonjomckay** [comment]
OK, thanks - I was hoping it would have been caused by the same issue, but I guess not! I'll look into it ASAP.

**Comment by jonjomckay** [comment]
I can't seem to figure out what's causing this error - I've had a bunch of bug report emails with the same message, but the stack trace is entirely useless, as there's no Fritter-specific context 😢:

```
#0 _GrowableList.single (dart:core-patch/growable_array.dart:342)
#1 ScrollController.position (package:flutter/src/widgets/scroll_controller.dart:110)
#2 RawScrollbarState._maybeTriggerScrollbar.<anonymous closure> (package:flutter/src/widgets/scrollbar.dart:862)
#3 SchedulerBinding._invokeFrameCallback (package:flutter/src/scheduler/binding.dart:1144)
#4 SchedulerBinding.handleDrawFrame (package:flutter/src/scheduler/binding.dart:1090)
#5 SchedulerBinding._handleDrawFrame (package:flutter/src/scheduler/binding.dart:998)
#6 _rootRun (dart:async/zone.dart:1354)
#7 _CustomZone.run (dart:async/zone.dart:1258)
#8 _CustomZone.runGuarded (dart:async/zone.dart:1162)
#9 _invoke (dart:ui/hooks.dart:161)
#10 PlatformDispatcher._drawFrame (dart:ui/platform_dispatcher.dart:253)
#11 _drawFrame (dart:ui/hooks.dart:120)
```

The only thing I can think of is if you're able to try running a debug build of Fritter, as that will have assertions enabled, which hopefully will error with the actual problem. I've enabled debug builds, so you should be able to download the APK from [GitHub Actions](https://github.com/jonjomckay/fritter/runs/2746973842). It'll appear as a separate application called Fritter (Debug), and you should be able to export from the release version, and import into the debug version.

**Comment by WindHun7er** [comment]
Don't worry, you're doing your best ! 😁
I sent you an email with what I could get... I hope it will help though...

**Comment by jonjomckay** [comment]
@WindHun7er you've probably already seen my [comment in the other issue](https://github.com/jonjomckay/fritter/issues/193#issuecomment-882570406), but this should be fixed in the upcoming v2.14.0, finally!
